code/go/0chain.net/smartcontract/dbs/benchmark/setup.go

```diff
@@ -125,7 +126,9 @@ func AddMockBlocks(
 				StateChangesCount:     33,
 				RunningTxnCount:       "mock running txn count",
 				RoundTimeoutCount:     0,
-				CreatedAt:             time.Now(),
+				UpdatableModel: model.UpdatableModel{
```

Comment:
do not create the structure here, simply setup the field

Reply:
Do you mean leaving "CreatedAt:   time.Now()" as it is? 
If that is the case, I changed it because it gave me an error.